api/app.py

```diff
@@ -11,28 +11,36 @@
     ParseFromDuckDB,
 )
 
+from shareable import MongoDBConnection, PlanData
+
 from loguru import logger
 
 app = FastAPI()
-con = None
-
+duckConn = None
+mongoConn = MongoDBConnection()
```

Comment:
Here, I am using the MongoDB connection using a state management instance, rather than accessing them via a global variable, we should consider that for DuckDB later as well.

Reply:
Created a PR for this
https://github.com/voltrondata/substrait-fiddle/pull/32